ledger/sandbox/BUILD.bazel

```diff
@@ -383,6 +383,23 @@ server_conformance_test(
     ],
 )
 
+server_conformance_test(
+    name = "conformance-test-transaction-stream-termination",
+    flaky = True,
+    server_args = [
+        "--static-time",
+    ],
+    servers = {
+        "h2database": SERVERS["h2database"],
+        "postgres": SERVERS["postgres"],
+        # this test is for bug #3984 that only happens with the SQL backend.
+        # the bug cannot occur in the in-memory backend by construction
+    },
+    test_tool_args = [
+        "--include=TransactionStaticTimeIT",
```

Comment:
You probably need to exclude this from `conformance-test-static-time` and `conformance-test-wall-clock`.

Reply:
I moved the test to sandbox specific tests.